tomcat/scripts/tomcat_entrypoint.sh

```diff
@@ -45,6 +45,15 @@ if [ -n "$TOMCAT_REVERSEPROXY_FQDN" ]; then
   echo "Info: ReverseProxy configuration is finished"
 fi
 
+if [ -n "$RELAXED_QUERY_CHARS" ]; then
+    mv $CATALINA_HOME/conf/server.xml $CATALINA_HOME/conf/server.xml.ori;
+    echo "Info: applying /docker/server.relaxedQueryChars.patch on $CATALINA_HOME/conf/server.xml"
+    patch -i /docker/server.relaxedQueryChars.patch -o $CATALINA_HOME/conf/server.xml $CATALINA_HOME/conf/server.xml.ori
+    sed -i -e "s|RELAXED_QUERY_CHARS|'$RELAXED_QUERY_CHARS'|g" $CATALINA_HOME/conf/server.xml;
```

Comment:
@torbrenner diese Zeile funktioniert leider nicht. Hast du ein Vorschlag ?

Reply:
Was an der Zeile funktioniert nicht? Steht in der server.xml am Ende nur `relaxedQueryChars="RELAXED_QUERY_CHARS"` oder fehlt der komplette Eintrag?
Wie verhält sich die Funktion im Zusammenspiel mit einem gesetzten Reverse Proxy? Könnte mir vorstellen, dass die Patchfiles da durcheinander kommen.

Reply:
> Was an der Zeile funktioniert nicht? Steht in der server.xml am Ende nur `relaxedQueryChars="RELAXED_QUERY_CHARS"` 

Ja genau. Die sed Befehlt ist schiefgelaufen. Hier ist die Fehlermeldung : 
**sed: -e expression #1, char 25: unknown option to `s'**

Es hat bis jetzt zusammen mit dem gesetzten Reverse Proxy funktioniert
```
<Connector port="8080" protocol="HTTP/1.1"
               relaxedQueryChars="RELAXED_QUERY_CHARS"
               connectionTimeout="20000"
               redirectPort="8443"
               proxyName="localhost"
               proxyPort="80"
               scheme="http"
               secure="false" />
```

Reply:
Der sed Befehl selbst ist syntaktisch richtig, das Einzige was mir einfällt, ist das in der Environment Variable auch das Zeichen "|" enthalten ist. Das würde vermutlich die Syntax zerstören.